### PR TITLE
add note about node <=0.10

### PR DESCRIPTION
**Pull request by fanatid**
Related #4 
I wasn't right about dropping 0.10 support or using shim.

### CODE REVIEW

**Comment by dcousens** [comment]
This isn't really any better?
The tests aren't failing,  but the code in the wild using `0.10` will.

**Comment by fanatid** [comment]
Hmm, you right, we should put note to README and add field engines to package.json

**Comment by fanatid** [comment]
Updated.

File: package.json [hunk 1]
```diff
@@ -30,5 +30,8 @@
     "nyc": "^6.1.1",
     "standard": "^6.0.8",
     "tape": "^4.5.1"
+  },
+  "engines": {
+    "node": ">0.10"
```
Comment (dcousens) [inline package.json:35]: Would this work for `0.10.1`?

Comment (fanatid) [inline package.json:35]: Good question. Answer: _no_.

``` js
$ node
> require('semver').satisfies('0.10.1', '>0.10')
false
> require('semver').satisfies('0.11.0', '>0.10')
true
```

**Comment by fanatid** [comment]
ping @dc

**Comment by fanatid** [comment]
@dc sorry!
actually wanted ping @dcousens

**Pull request merged**